gravitee-apim-gateway/gravitee-apim-gateway-standalone/gravitee-apim-gateway-standalone-container/pom.xml

```diff
@@ -32,7 +32,7 @@
 
     <properties>
         <jetty94.wiremock.version>9.4.44.v20210927</jetty94.wiremock.version>
-        <gravitee-connector-http.version>1.1.10</gravitee-connector-http.version>
+        <gravitee-connector-http.version>1.1.11</gravitee-connector-http.version>
```

Comment:
in 3.18.x and upper, we should use the 2.0.x branch of connector-http.
See https://github.com/gravitee-io/gravitee-api-management/pull/2395

Reply:
Nice catch, I cherry-picked both bump commits :+1: